todo_app/auth/decorators.py

```diff
@@ -0,0 +1,14 @@
+from functools import wraps
+from flask_login import current_user
+
+from todo_app.auth.user_roles import UserRoles
+
+
+def requireWriter(func):
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        if current_user.is_anonymous or current_user.role == UserRoles.WRITER:
```

Comment:
This works perfectly here, but worth bearing in mind how much you'd trust this code in a production app - would you be content knowing that accidentally setting `LOGIN_DISABLED` in prod could allow anyone to write? Could we put any processes to stop that? Or could we use more code checks?
As implied, I don't think it's important/worthwhile making many changes here, but worth acknowledging the risk!